users/tests/test_models.py

```diff
@@ -30,3 +30,21 @@ def test_guardian_with_children_creation(project):
     assert Guardian.objects.count() == 1
     assert Child.objects.count() == 3
     assert list(Guardian.objects.first().children.all()) == list(Child.objects.all())
+
+
+@pytest.mark.django_db
+def test_guardian_email_populating():
+    user = UserFactory(email="user@example.com")
+
+    guardian = Guardian.objects.create(user=user)
+    assert guardian.email == "user@example.com"
+
+    guardian.email = "guardian.example.com"
```

Comment:
Is this a valid email address? I guess it shouldn't pass the validator

Reply:
Haha, good catch. Actually, the validation isn't run on model level, so if an email is set outside of admin UI and API, it is not validated. But I fixed the email to be a valid one so that the test isn't so confusing 🙂 

Whether the validation should be on model level is another question, perhaps, but I think that is out of the scope of this PR anyway, as other validations in Kukkuu work like this and should be all changed if we decide to do that.

Reply:
Yeah, I knew that kukkuu validation doesn't run on model, but since it's an [EmailField](https://docs.djangoproject.com/en/3.0/ref/models/fields/#emailfield) so I guess there will be at least some validation there. Not sure why it didn't catch this :D

Reply:
Yes one would assume that, but actually no validations are run by default, not even for EmailFields. As my previous test proved :D

Reply:
Yah, just checked a bit, I was wrong. The model `.save()` indeed doesn't trigger any validation.